pyproject.toml

```diff
@@ -46,11 +47,11 @@ max-complexity = 20
 max-line-length = 140
 
 extend-ignore = [
```

Comment:
Just formatting noise here. I'll revert.

Reply:
Why tho, let it. Whatever, not terrible.